lombok.config

```diff
@@ -16,3 +16,5 @@
 #
 
 lombok.addLombokGeneratedAnnotation = true
+
+lombok.log.fieldname=LOGGER
```

Comment:
Why we don't keep the default field name `log`?

Reply:
Lombok will generate a `static final ` field, so I changed the field name to uppercase. Is this inappropriate? Need to change to lowercase？
```java
@Generated
private static final Logger LOGGER = LoggerFactory.getLogger(Xxx.class);
```

> Why we don't keep the default field name `log`?

Reply:
I just think keep the default field name is better, since we already use `@Slf4j` in some class, this will reduce the change.